webapp/ofbiz-tutorial/WEB-INF/web.xml

```diff
@@ -52,7 +52,7 @@ under the License.
         <filter-class>org.apache.ofbiz.webapp.control.ControlFilter</filter-class>
         <init-param>
             <param-name>allowedPaths</param-name>
-            <param-value>/error:/control:/select:/index.html:/index.jsp:/default.html:/default.jsp:/images</param-value>
+            <param-value>/error:/control:/select:/index.html:/index.jsp:/default.html:/default.jsp:/images:/usermgr/css:/usermgr/js:/usermgr/includes</param-value>
```

Comment:
do we really need to change in to web.xml

Reply:
To include the external CSS and JS we were required to make changes in the web.xml to allow access to those resources.